README.md

```diff
@@ -34,6 +35,8 @@ NOTE: Coverage will not show in the analysis unless you have already generated t
 
 ## Intellij Version Control for external package
 
+IMPORTANT: Ensure ssh was used to install this repository as an external package in your current repository to be able to push changes. 
```

Comment:
## Working with both packages in Intellij

It is possible to edit the external package when using it in another repository, like nhais or mhs, in the same IntelliJ window as the opened project. There is no need to keep two windows open.

To do this you must ensure ssh was used to install this repository as an external package in your current repository to be able to push changes.